clang-tidy/action.yaml

```diff
@@ -60,7 +60,7 @@ runs:
 
     - name: Run rosdep install
       run: |
-        package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }})
+        package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }} --base-paths dependency_ws .)
```

Comment:
[Nits] `. dependency_ws` looks better to me because `dependency_ws` is optional. But since it makes no difference, whether to change it is up to you.

Reply:
@kenji-miyake fixed at https://github.com/autowarefoundation/autoware-github-actions/pull/76/commits/cb7932c2e32bbd5e7ced9d0dd1a5798a63a59b17.